sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -157,7 +195,23 @@ def plot(self, ax=None, *, name=None, **kwargs):
         ylabel = "Precision" + info_pos_label
         self.ax_.set(xlabel=xlabel, ylabel=ylabel)
 
-        if "label" in line_kwargs:
+        if plot_chance_level and self.prevalence_pos_label is not None:
+            chance_level_line_kw = {
+                "label": f"Chance level (AP = {self.prevalence_pos_label:0.2f})",
+                "color": "k",
+                "linestyle": "--",
+            }
+            if chance_level_kw is not None:
+                chance_level_line_kw.update(chance_level_kw)
+            (self.chance_level_,) = self.ax_.plot(
+                (0, 1),
+                (self.prevalence_pos_label, self.prevalence_pos_label),
+                **chance_level_line_kw,
+            )
+        else:
+            self.chance_level_ = None
```

Comment:
Did you think about making it an error to set `plot_chance_level` and not provide `prevalence_pos_label`?

The reason I'm asking is that it having to pass the right thing in two different locations (constructor and `plot`) to get the chance level line to display could lead to confused users. I was wondering if there is ever a reason to set `plot_chance_level=True` without also providing the prevalence? If there is no good reason to do that we could make it an error here?